simple/deployment.yaml

```diff
@@ -0,0 +1,3479 @@
+#kubernetes namespaces
```

Comment:
@NishikaDeSilva Add license header.

Reply:
is there anything to be changed in the license header i.e: year ?